src/python/pants/backend/python/lint/ruff/rules.py

```diff
@@ -28,23 +30,44 @@ class RuffFixRequest(FixTargetsRequest):
     field_set_type = RuffFieldSet
     tool_subsystem = Ruff
     partitioner_type = PartitionerType.DEFAULT_SINGLE_PARTITION
+    enable_lint_rules = False
```

Comment:
This deserves a comment 😄 

In fact, I have a suggestion. What if we moved `RuffFixRequest` below `RuffLintRequest`. Then set this flag AND assign `tool_id` to `RuffLintRequest`'s `tool_id`? And slap a comment above.

```python
class RuffLintRequest(...):
    ...
    
class RuffFixRequest(...):
    ...
    
    
    # NB: (blah blah blah)
    enable_lint_rules = False
    tool_id = RuffLintRequest.tool_id  # We have to use the same tool_id
```

(assuming the `tool_id` trick doesn't blow up anything)

What do you think?